install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -53,10 +53,12 @@ spec:
           - --monitoringAddr=:{{ .Values.global.monitoringPort }}
           - --domain
           - {{ .Values.global.proxy.clusterDomain }}
-{{- if .Values.global.oneNamespace }}
-          - "-a"
-          - {{ .Release.Namespace }}
-{{- end }}
+          - --appNamespaces
```

Comment:
put that inside the block - no need to pass params if the value is the default 
( problems on upgrade/downgrade,  noise, consistency, etc)

Reply:
done.